pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -245,6 +262,8 @@ buildStdenv.mkDerivation ({
     export BUILD_OFFICIAL=1
   '');
 
+  configurePlatforms = [ ];
```

Comment:
Why is this needed?

Reply:
https://github.com/NixOS/nixpkgs/blob/640239d7d6414d98c0ab286d76d6bd8962df938b/pkgs/applications/networking/browsers/firefox/common.nix#L285

Reply:
Should add a comment to explain, otherwise it'll end up being removed by accident, see https://github.com/NixOS/nixpkgs/pull/152715